scripts/github-actions/generate.sh

```diff
@@ -60,6 +60,8 @@ for tag in $tags; do
 				os: (
 					if (.constraints | contains(["windowsservercore-1809"])) or (.constraints | contains(["nanoserver-1809"])) then
 						"windows-2019"
+					elif (.constraints | contains(["windowsservercore-ltsc2022"])) or (.constraints | contains(["nanoserver-ltsc2022"])) then
+						"windows-2022"
```

Comment:
Nice!

I think I'd like to swap these aesthetically, though (so the list stays in descending order):

```suggestion
					if (.constraints | contains(["windowsservercore-ltsc2022"])) or (.constraints | contains(["nanoserver-ltsc2022"])) then
						"windows-2022"
					elif (.constraints | contains(["windowsservercore-1809"])) or (.constraints | contains(["nanoserver-1809"])) then
						"windows-2019"
```